elements/rh-band/src/rh-band.scss

```diff
@@ -5,12 +5,15 @@ $SLOT: "";
 $aside-md: 250px;
 $aside-lg: 300px;
 
+
+
 :host {
-  // Spacing values
-  --rh-band--paddingTop:                      calc(#{rh-var(container-spacer)} * 4);
-  --rh-band--paddingRight:                    #{rh-var(container-spacer)};
-  --rh-band--paddingBottom:                   calc(#{rh-var(container-spacer)} * 4);
-  --rh-band--paddingLeft:                     #{rh-var(container-spacer)};
+  // Default spacing values
+  --rh-band--padding-vertical:                 calc( #{rh-var(container-spacer)} * 4);
```

Comment:
I think this is a case where we would use the underscores for the naming: `--rh-band--padding_vertical`

Reply:
sure, updated